### ISSUE TITLE
Accessibility: The user is unable to toggle checkbox acceptance using the keyboard

### ISSUE THREAD

**Issue by AndrewBarber** [bigcommerce/checkout-js#1724]
On checkout, the user can hit the tab key to highlight the “Subscribe to our newsletter” checkbox, on the first step of checkout.

If the user hits the enter key, this does not tick / untick the checkbox but will instead progress the user to the next step of checkout - as it appears the ‘Continue’ CTA is selected.

Additionally, the user cannot toggle the “My billing address is the same as my shipping address” checkbox on the shipping step in the same manner. This issue also affects the T&C’s checkbox at the end of checkout, where the user cannot accept the T&C’s using the enter key

The user should be able to use the tab and enter keys to toggle acceptance of checkboxes within checkout.
